srcpkgs/mpd/template

```diff
@@ -37,7 +37,7 @@ license="GPL-2.0-or-later"
 homepage="https://www.musicpd.org/"
 changelog="https://raw.githubusercontent.com/MusicPlayerDaemon/MPD/v${version}/NEWS"
```

Comment:
```suggestion
changelog="https://raw.githubusercontent.com/MusicPlayerDaemon/MPD/master/NEWS"
```

Reply:
I think it's because musl strptime does not support the `%F` field descriptor